test/performance/latency_test.go

```diff
@@ -0,0 +1,78 @@
+// +build performance
+
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// latency_test.go brings up a helloworld app and gets the latency metric
+
+package performance
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/knative/pkg/test/logging"
+	"github.com/knative/pkg/test/spoof"
+	"github.com/knative/serving/test"
+	"github.com/knative/test-infra/shared/testgrid"
+)
+
+func TestPerformanceLatency(t *testing.T) {
```

Comment:
@srinivashegde86 , may I suggest to rename the test to something more specfic, such as TestTimeToServeLatency?